backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/segmentedcontrol/internal/BpkSegmentedControlColors.kt

```diff
@@ -0,0 +1,35 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.segmentedcontrol.internal
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import net.skyscanner.backpack.compose.utils.dynamicColorOf
+
+internal val surfaceContrastSegmentedControlOn: Color
```

Comment:
where do these come from? there's already a `BpkSegmentedControlColors` file which is the custom colours which i think should be used here?

Reply:
noice, didn't notice that till I rebased 😓

Reply:
actually they were there even before 😅